shotover-proxy/examples/windsock/cassandra.rs

```diff
@@ -99,9 +192,21 @@ impl Bench for CassandraBench {
                     Topology::Cluster3 => "cluster3".to_owned(),
                 },
             ),
-            self.shotover.to_tag(),
```

Comment:
I think this was lost in a merge conflict but this `to_tag` method is a helper that handles the shotover tag for us.